src/app/about/page.tsx

```diff
@@ -16,13 +120,21 @@ export default function About() {
           careers in software development.
         </p>
       </div>
-      <Image
-        className={styles.image}
-        src="fall_2022_planning_meeting.jpg"
-        alt="SSD members (about 30) at our Fall 2022 planning meeting in CISE."
-        height={600}
-        width={600}
-      />
+      <div className={styles.imageContainer}>
+        <Image
+          className={styles.image}
+          src="fall_2022_planning_meeting.jpg"
+          alt="SSD members (about 30) at our Fall 2022 planning meeting in CISE."
+          height={600}
+          width={600}
+        />
+      </div>
+      <section className={styles.officers}>
+        <div className={styles.container}>
+          <h2>Officers</h2>
```

Comment:
[Please Change]: This should be centered.
![image](https://github.com/ufssd/ufssd-website/assets/35618116/2b352824-5efb-48d2-9c9d-e2b29568c5cf)